lambda_src/response_storage.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Dict, Text
+
+import boto3
+
+AWS_REGION = os.environ[
```

Comment:
put on one line (use black), and we might want to set a default when the envar is not present

Reply:
Ah okay 👍